pages/tags/[tag].tsx

```diff
@@ -16,14 +16,14 @@ interface Props {
 }
 
 export const getStaticPaths: GetStaticPaths = async () => {
-    const posts = await loadAllPosts()
+    const posts = await loadAllRecords('posts')
```

Comment:
Just wanted to surface that we likely will want to add tags to guides (unless that was something discussed), and right now this file is only concerned with `posts`.

Reply:
Thanks for pointing this out! Meant to leave a note about it, as I was *fairly* sure we would want to include guides here, but not 100% sure. Just added a commit that should give us that flexibility. I'm calling everything `records`, but open to suggestions! We could also do `Posts and guides tagged with...` or something like that.

Reply:
Thanks for updating this!